temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflowContext.java

```diff
@@ -416,10 +495,16 @@ private ExecuteLocalActivityParameters constructExecuteLocalActivityParameters(
         toRetryPolicy(RetryOptions.newBuilder(retryOptions).validateBuildWithDefaults()));
     Duration localRetryThreshold = options.getLocalRetryThreshold();
     if (localRetryThreshold == null) {
-      localRetryThreshold = replayContext.getWorkflowTaskTimeout().multipliedBy(6);
+      localRetryThreshold = replayContext.getWorkflowTaskTimeout().multipliedBy(3);
```

Comment:
No. An arbitrary number is less than 6 because the meaning of this parameter changed from the duration of retries to a duration of a single timer. At the same time, I don't want to keep it close to 1 because of the potential accuracy issues that we discussed.